google/cloud/opentelemetry/quickstart/.bazelrc

```diff
@@ -30,5 +30,15 @@ build:macos --host_cxxopt=-std=c++14
 build --experimental_convenience_symlinks=ignore
 
 # Enable OpenTelemetry tracing instrumentation for google-cloud-cpp.
-build --@io_opentelemetry_cpp//api:with_abseil
 build --@google_cloud_cpp//:enable_opentelemetry
+
+# OpenTelemetry versions < v1.16.0 are not compatible with Abseil
```

Comment:
After me, nobody will read this comment block.  `doc/packaging.md` may be a better place, or the `CHANGELOG.md` file.

Reply:
>  nobody will read this comment block.

Likely true. I was really banking on people finding the write up in the quickstart's README.

> `doc/packaging.md` may be a better place, or the `CHANGELOG.md` file.

You are probably right that the more places we say words the better. I wrote up some words in the CHANGELOG. I also added a link (if you click enough times) from the packaging doc to the quickstart README.